.github/workflows/builds.yml

```diff
@@ -37,6 +45,7 @@ jobs:
           ${SETUP} &&
           cmake -B build -S .
           -GNinja
+          -DCMAKE_CXX_COMPILER_LAUNCHER=$(find / -type f -name "ccache")
```

Comment:
would `which ccache` do the job too?

Reply:
or just `ccache` like below

Reply:
In the CentOS7/8 + LCG images, ccache is installed in a funky location somewhere. I realize now though, that after the LCG view setup script, it is actually in `$PATH`, so `ccache` should be enough. I'll change that.